### DISCUSSION TITLE
Zoom the sidebar

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#788] [Ideas]
### What feature would you like to see?

I'd like to have the possibility to zoom the sidebar, couse on a big monitor it is very small.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/597_

**Comment by trendespresso** [comment]
I'd like to be able to zoom out the sidebar because on my 13" Macbook the font is too large. Just the standard Cmd/Ctrl and + or - buttons would be great!

**Comment by mikedlr** [comment]
I've +1ed this (being a good boy, I did it with the upvote icon on the original report -  this is not just a +1 spam comment.)

I've also noticed that there are two related Reddit Posts
* [Icon Badges and Sidebar Scaling ](https://www.reddit.com/r/zen_browser/comments/1g0npn6/icon_badges_and_sidebar_scaling/) 
* [Zen sidebar tab. How can I change the sidebar width?. Right now it's possible to change maximum width but the minimum it can't be changed. When expanded there should be the option to change it to any width.](https://www.reddit.com/r/zen_browser/comments/1fno13p/zen_sidebar_tab_how_can_i_change_the_sidebar/)

so I'm adding the links here for cross reference and have already added them on the other side back to here.

**Comment by mikedlr** [comment]
BTW, workaround for now for making bigger icons.

* go to about:config 
* set layout.css.devPixelsPerPx to 2

everything in the browser is now double scaled. Some icons look ugly.

**Reply by syoxTS** [reply]
Just trying out the browser for the first time, this is great but it scales everything, like you said. Has there been any update on having this exact function but ONLY for the sidebar? Thank you!

**Reply by nonvenomous** [reply]
Note this causes issues with multiple monitors that have different pixel ratios.

**Comment by SirSova** [comment]
Being able to customize tab spacing, and font size (and icons accordingly) would be incredibly useful! 
Tabs look too small on a high-resolution screen. Adaptive scaling would be the best solution) but I have no idea how difficult it can be...

**Reply by mikedlr** [reply]
Did you try my workaround? Did it give you what you want (but without the easy interface to change the scaling?)

**Comment by mikedlr** [comment]
If more space is needed in pinned tabs, the [SuperPins mod](https://zen-browser.app/mods/ad97bb70-0066-4e42-9b5f-173a5e42c6fc/) can be useful. It reduces the pinned icons from a full line to just wider than an icon. That means that several pinned tabs can fit on one line.
